front/chat/src/Components/ChatMessageForm.svelte

```diff
@@ -376,20 +441,28 @@
                 >
                     <SmileIcon size="17" />
                 </button>
-                <input type="file" id="file" name="file" class="tw-hidden" on:input={handleInputFile} multiple />
-                <label for="file" class="tw-mb-0 tw-cursor-pointer"><PaperclipIcon size="17" /></label>
+                {#if ENABLE_CHAT_UPLOAD || ADMIN_API_URL}
```

Comment:
We need to be able to disable the chat uploads for some clients, from the admin.

So ENABLE_CHAT_UPLOAD is good as an env var, but in case there is an admin, we should use the value returned during the call to /room/access (so we should add a "enableChatUpload: boolean" property to "/room/access"

Reply:
Made in PR #2598